libs/web/settings/data-access/src/lib/store/settings.reducer.ts

```diff
@@ -1,15 +1,14 @@
 import { createReducer, on } from '@ngrx/store';
 import * as SettingsActions from './settings.actions';
+import { LocalStorageService } from '../services';
 
 export const SETTINGS_FEATURE_KEY = 'settings';
 
 export interface SettingsState {
   volume: number;
 }
 
-export const initialState: SettingsState = {
-  volume: 0
-};
+export const initialState: SettingsState = LocalStorageService.loadInitialState()?.settings ?? {}
```

Comment:
FYI I skipped here the `volume: 0` since the `getSettingsVolume` selector defaults to 0 already.